patient-portal-backend/src/main/resources/application.properties

```diff
@@ -1,6 +1,2 @@
-server.compression.enabled=true
```

Comment:
I moved it into the pom of where the PatientDashBoard application is loaded. On startup, Spring Boot wasn't picking up my DataSource configurations in the child pom's .properties file so moved it up. Decided to move this up as well so that we're only maintaining one properties file.